### PR TITLE
pr_21

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 3 issues

File: src/main/java/org/sonarqube/A.java [hunk 1]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAS4yv3CSWYuT4mbB7f###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_21@9
A "NullPointerException" could be thrown; "message" is nullable here.

Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAS4yv3CSWYuT4mbB7e###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_21@9
The return value of "length" must be used.

File: src/main/java/org/sonarqube/A.java [hunk 2]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java]: ###AWAS4yv3CSWYuT4mbB7d###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_21@6
Complete the task associated to this TODO comment.

Comment (sns-seb) [reply src/main/java/org/sonarqube/A.java]: simulating user comment on about to be deleted comment

Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java]: simulating bot comment not autogenerated

Comment (sns-seb) [reply src/main/java/org/sonarqube/A.java]: let's do another user comment, just because

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 2 issues

File: src/main/java/org/sonarqube/A.java (hunk 1 above)
Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java:11]: ===RESOLVED===
This issue has been resolved by a user in SonarQube

**Review by sns-seb2 — APPROVED** [review]

File: src/main/java/org/sonarqube/A.java (hunk 1 above)
Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java:11]: ===RESOLVED===
This issue has been resolved by a user in SonarQube

**Pull request closed without merging**